docs/supported-technologies.asciidoc

```diff
@@ -38,6 +40,31 @@ If you are interested in accurately measuring the duration of load event, the in
 `Fire load event` Span or from the Transaction marks available as `transaction.marks.agent.domComplete` in the Elasticsearch document.
 
 
+[float]
+[[user-centric-metrics]]
+=== User-centric Metrics
+
+To understand the performance characteristics of a web page beyond the page load and how the user perceives performance, the agent supports capturing these responsiveness metrics.
+
+`First contentful paint (FCP)`::
+Focusses on the initial rendering and measures the time from when the page starts loading to when any part of the page's content is displayed on the screen. Captured as transaction mark for `page-load` transaction.
+
+`Largest contentful paint (LCP)`::
```

Comment:
The same with the capital letter since it's abbreviated

Reply:
I just changed everything based on @bmorelli25 suggestion 

> We try to use sentence case in the docs, so I've added suggestions for that below.